simtools/configuration/commandline_parser.py

```diff
@@ -197,20 +197,20 @@ def initialize_telescope_model_arguments(self, add_model_version=True, add_teles
 
         Parameters
         ----------
-        add_model_version: str
-            Model version.
-        add_telescope: str
-            Telescope model name (e.g. LST-1, SST-D, ...).
+        add_model_version: bool
+            Set to allow a simulation model argument.
+        add_telescope: bool
+            Set to allow a telescope name argument.
         """
 
         _job_group = self.add_argument_group("telescope model")
         _job_group.add_argument(
-            "--site", help="CTAO site (e.g. North, South)", type=self.site, required=False
+            "--site", help="CTAO site (e.g., North, South)", type=self.site, required=False
```

Comment:
North, South are examples or the only options?

Reply:
It can actually be paranal, south, lapalma or north. I don't think we should list all of these options, the error message would be clear if the user makes a mistake.